options/options.go

```diff
@@ -14,23 +15,35 @@ type Options struct {
 	DryRun    bool
 	ParseTime bool
 	Config    *string
+	Edit      *string
 }
 
 var OptionsToPrint = `
   -h, --help              Prints this help message
-  -c, --config            Reads the config from custom path
+  -c, --config [path]     Reads the config from custom path
   -d, --dry-run           Prints bindings and their commands and exits
   -k, --kill	          Gracefully kills every running instances of dxhd
   -p, --parse-time        Prints how much time parsing a config took
   -r, --reload	          Reloads every running instances of dxhd
-  -v, --version	          Prints current version of program`
+  -v, --version	          Prints current version of program
+  -e, --edit [file]	      Shortcut to edit a file in dxhd's config folder. Opens dxhd.sh if file is empty.`
 
 func Parse() (opts Options, err error) {
 	osArgs := os.Args[1:]
 
 	skip := false
 
-toplevel:
+	notEnoughArgsErr := errors.New("Not enough arguments given")
```

Comment:
The same as the review of line 101.

Reply:
There's no comment on line 101 in this file. There is in main.go but it doesn't seem relevant to this?

Reply:
Ah, right. The "review mode" showed me the main.go file first, and I just assumed it was it. Anyway, yeah, it's relevant (the error message should be all lowercase)

Reply:
But just do this instead https://github.com/dakyskye/dxhd/pull/29#discussion_r503098222